charts/seqr/templates/deployment.yaml

```diff
@@ -53,6 +53,12 @@ spec:
         resources:
           {{- toYaml .Values.resources | nindent 10 }}
         {{- include "seqr.environment" $ | nindent 8 }}
+        livenessProbe:
```

Comment:
We've seen some strange behavior recently with the seqr service failing to come up correctly during a re-spin.

My best guess at what's been happening here: 
- our container runs regardless of whether or not the backgrounded server is actually running or not (`sleep 100000`)
- gunicorn never comes up successfully if `gcloud` or something else `init`-related is failing.  Testing this locally showed that only a single worker failing will bring down all N worker processes.  
- the readiness probe never succeeds on that pod/
- we can end up in a serving-traffic-correctly state if we have multiple pods running (1 running correctly and 1 not running).  
- we can also end up in a fully down state if both pods happen to have an init issue. I'm not sure this can happen during a deployment (the internet points to the rolling deploy mechanism using "readiness" rather than liveness), but it has happened during node swap-outs.  

Adding the livenessProbe will help k8s know the container is DOA and should be kicked.  

This got mentioned in slack many months ago: https://the-tgg.slack.com/archives/C01FRFB36LR/p1682518495987779